benchmarks/benchmark_down_sample_sampler.py

```diff
@@ -0,0 +1,303 @@
+# Write the benchmarking functions here.                                        
+# See "Writing benchmarks" in the asv docs for more information.
+
+import os
+
+from magellan.sampler.down_sample import down_sample
+from magellan.utils.generic_helper import get_install_path
+from magellan.io.parsers import read_csv_metadata
+
+import sys
+if sys.version[0] == '2':
+    reload(sys)
+    sys.setdefaultencoding("utf-8")
+
+PATH = get_install_path()
+DATASET_PATH = os.sep.join([PATH, 'datasets', 'example_datasets'])
+
+class TimeDownSampleRestaurants:
+    def time_down_sample_tables(self):
+        path_for_a = os.sep.join([DATASET_PATH, 'restaurants', 'A.csv'])
+        path_for_b = os.sep.join([DATASET_PATH, 'restaurants', 'B.csv'])
+        self.size = 500
+        self.y_param = 2
+        try:
+            self.A = read_csv_metadata(path_for_a)
+            self.B = read_csv_metadata(path_for_b)
```

Comment:
Time taken to read the tables will also get included in the down sampling time. You should separate the data loading time.